File: frontend/app/src/main/java/com/example/commov/model/ProjectMember.java
package com.example.commov.model;

import androidx.annotation.ColorRes;

public final class ProjectMember {
    public final String name;
    public final String initials;
    @ColorRes
    public final int avatarColorResId;

    public ProjectMember(String name, String initials, @ColorRes int avatarColorResId) {
        this.name = name;
        this.initials = initials;
        this.avatarColorResId = avatarColorResId;
    }
}

File: frontend/app/src/main/java/com/example/commov/model/ProjectMember.kt
package com.example.commov.model

import androidx.annotation.ColorRes

data class ProjectMember(
    val userId: Int = 0,
    val name: String,
    val initials: String,
    @ColorRes val avatarColorResId: Int,
    val isManager: Boolean = false,
    val photo: String = ""
)
